frontend/src/entities/notebook/notebook.actions.tsx

```diff
@@ -198,7 +198,7 @@ const NotebookActionHandler = async (
             `${success ? 'Successfully' : 'Failed to'} ${e.detail.id}${
                 e.detail.id === 'stop' ? 'p' : ''
             }${success ? 'ing' : ''} notebook instance ${notebook.resourceId}${
-                success ? '.' : ` because: ${response.data}`
+                success ? '.' : ` because: ${response.payload}`
             } `,
```

Comment:
I see this message isn't your doing but this error message is incredibly unreadable IMHO. I think we should break out some of this inline logic into their own variables (or just use an if-statement for success or not) so this string because easier to visually parse. If you agree lets also stop checking success in such a manual way and instead use `isFulfilled` ([link](https://github.com/awslabs/mlspace/blob/main/frontend/src/entities/batch-translate/create/batch-translate-create.tsx#L203) to how it is used in our code already).